src/shared/actions/alerts.js

```diff
@@ -89,6 +89,17 @@ export const generateNodeOutOfSyncErrorAlert = () => (dispatch) => {
     dispatch(generateAlert('error', i18next.t('global:nodeOutOfSync'), i18next.t('global:nodeOutOfSyncExplanation')));
 };
 
+/**
+ * Generates an error alert if a network call is made to an unsupported node
+ *
+ * @method generateUnsupportedNodeErrorAlert
+ *
+ * @returns {function} dispatch
+ */
+export const generateUnsupportedNodeErrorAlert = () => (dispatch) => {
+    dispatch(generateAlert('error', i18next.t('global:unsuppoortedNode'), i18next.t('global:unsuppoortedNodeExplanation')));
```

Comment:
This should be changed to match the names of the strings in the [translation.json](https://github.com/iotaledger/trinity-wallet/pull/805/files#diff-8448e7037db660ba08a9b7c6efaae24a)
```suggestion
    dispatch(generateAlert('error', i18next.t('global:experimentalNode'), i18next.t('global:experimentalNodeExplanation')));
```